[2025-09-26T00:02:20.357+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-26T00:02:20.365+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.run_forecast scheduled__2025-09-25T00:00:00+00:00 [queued]>
[2025-09-26T00:02:20.367+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_forecast_stock_robust.run_forecast scheduled__2025-09-25T00:00:00+00:00 [queued]>
[2025-09-26T00:02:20.367+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-09-26T00:02:20.371+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_forecast> on 2025-09-25 00:00:00+00:00
[2025-09-26T00:02:20.374+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=17993) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-26T00:02:20.375+0000] {standard_task_runner.py:64} INFO - Started process 17995 to run task
[2025-09-26T00:02:20.375+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ml_forecast_stock_robust', 'run_forecast', 'scheduled__2025-09-25T00:00:00+00:00', '--job-id', '414', '--raw', '--subdir', 'DAGS_FOLDER/ml_forecast_stock.py', '--cfg-path', '/tmp/tmppxwp2k6j']
[2025-09-26T00:02:20.376+0000] {standard_task_runner.py:91} INFO - Job 414: Subtask run_forecast
[2025-09-26T00:02:20.394+0000] {task_command.py:426} INFO - Running <TaskInstance: ml_forecast_stock_robust.run_forecast scheduled__2025-09-25T00:00:00+00:00 [running]> on host 11e1994bdecf
[2025-09-26T00:02:20.421+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data226' AIRFLOW_CTX_DAG_ID='ml_forecast_stock_robust' AIRFLOW_CTX_TASK_ID='run_forecast' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T00:00:00+00:00'
[2025-09-26T00:02:20.422+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-26T00:02:20.437+0000] {base.py:84} INFO - Using connection ID 'snowflake_default' for task execution.
[2025-09-26T00:02:20.438+0000] {connection.py:408} INFO - Snowflake Connector for Python Version: 3.11.0, Python Version: 3.12.4, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-09-26T00:02:20.438+0000] {connection.py:1258} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-09-26T00:02:21.268+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:21.357+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:21.538+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:21.539+0000] {ml_forecast_stock.py:223} INFO - Using: warehouse="BEETLE_QUERY_WH", database="ASH_DB", schema="STOCK_SCHEMA"
[2025-09-26T00:02:21.660+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:21.661+0000] {ml_forecast_stock.py:231} INFO - Executing SQL statement 1/2
[2025-09-26T00:02:49.597+0000] {cursor.py:1149} INFO - Number of results in first chunk: 35
[2025-09-26T00:02:49.600+0000] {ml_forecast_stock.py:231} INFO - Executing SQL statement 2/2
[2025-09-26T00:02:51.166+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:51.254+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-09-26T00:02:51.255+0000] {ml_forecast_stock.py:237} INFO - All SQL statements executed successfully
[2025-09-26T00:02:51.255+0000] {connection.py:773} INFO - closed
[2025-09-26T00:02:51.319+0000] {connection.py:779} INFO - No async queries seem to be running, deleting session
[2025-09-26T00:02:51.693+0000] {ml_forecast_stock.py:332} INFO - Forecast executed successfully
[2025-09-26T00:02:51.695+0000] {python.py:237} INFO - Done. Returned value was: success
[2025-09-26T00:02:51.695+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-26T00:02:51.716+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ml_forecast_stock_robust, task_id=run_forecast, run_id=scheduled__2025-09-25T00:00:00+00:00, execution_date=20250925T000000, start_date=20250926T000220, end_date=20250926T000251
[2025-09-26T00:02:51.769+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-26T00:02:51.784+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-26T00:02:51.785+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
